api/v1alpha1/dataprotectionapplication_types.go

```diff
@@ -348,11 +348,19 @@ type NodeAgentCommonFields struct {
 	// timeout defines the NodeAgent timeout, default value is 1h
 	// +optional
 	Timeout string `json:"timeout,omitempty"`
+	// How long to wait for preparing a DataUpload/DataDownload. Default is 30 minutes.
```

Comment:
this should be added to restic as well?

Reply:
you are right, it's regardless of the uploader type, will change

Reply:
no no, it is currently being added to restic as well

but we pretend to remove restic usage soon https://github.com/openshift/oadp-operator/pull/1235

should this just be added to node agent config struct?

Reply:
Ah got it, no I don't think so, we should not add the new items to the Restic config options imo, we don't want users to use Restic ones anymore, right?